Plugins/Flow.Launcher.Plugin.WebSearch/plugin.json

```
@@ -26,7 +26,7 @@
   "Name": "Web Searches",
   "Description": "Provide the web search ability",
   "Author": "qianlifeng",
-  "Version": "1.4.0",
+  "Version": "1.4.1",
```

no actual changes with just a white space, so no need for bumping the version 
```suggestion
  "Version": "1.4.0",
```

---

> no actual changes with just a white space, so no need for bumping the version

I'm sorry. I didn't know it.

---

All good, no worries